### DISCUSSION TITLE
How to enable a node in main.

### DISCUSSION THREAD

**Discussion by VinayShirol-eaton** [zephyrproject-rtos/zephyr#54477] [Q&A]
Hi,

I want to use LSE clock of the board. Because of few reasons I need to disable the node in overlay file, but I need to enable the node in main. 

Is this possible to achieve? If yes, please suggest me how?

Thanks

**Comment by erwango** [comment]
Not possible using Zephyr API.
You need to use STM32Cube LL API.

**Comment by VinayShirol-eaton** [comment]
Thanks for your reply.

Could you suggest me the LL Api that I can use here ?

**Reply by erwango** [reply]
Here is a piece of code that should help, but for more information, I suggest to study STM32Cube packages examples:
```
		LL_RCC_LSE_Enable();
		while (!LL_RCC_LSE_IsReady()) {
			/* Wait for LSE ready */
		}
```
